database/interface.py

```diff
@@ -187,9 +194,12 @@ def delete_harvest_job(self, job_id):
         self.db.commit()
         return "Harvest job deleted successfully"
 
-    def add_harvest_error(self, error_data):
+    def add_harvest_error(self, error_data: dict, error_type: str):
```

Comment:
probably want a specific response for not passing `error_type` as that's going to be easy enough to forget now.

Reply:
added [here](https://github.com/GSA/datagov-harvesting-logic/blob/8baa21616de9693b43ff78e40f8e05d0cfeba56e/database/interface.py#L199)